chainer/functions/noise/dropout.py

```diff
@@ -63,7 +63,9 @@ def dropout(x, ratio=.5, **kwargs):
        See :func:`chainer.using_config`.
 
     Args:
-        x (~chainer.Variable): Input variable.
+        x (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`): Input variable. \
+        A :math:`(s_1, s_2, ..., s_N)` -shaped float array.
```

Comment:
The comment is ok.  But it seems to make some indent error. You do not need to put the backslash after the "Input variable" and have to put the indent before "A math". Please see detail if https://github.com/chainer/chainer/blob/master/chainer/functions/activation/sigmoid.py